cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -988,6 +1020,133 @@ define(["jquery", "common/js/spec_helpers/ajax_helpers", "common/js/components/u
                     expect($("#id_time_limit").val()).toBe("02:30");
                 });
 
+                it('can select prerequisite', function() {
+                    createCourseOutlinePage(this, mockCourseJSON, false);
+                    outlinePage.$('.outline-subsection .configure-button').click();
+                    selectPrerequisite();
+                    $('.wrapper-modal-window .action-save').click();
```

Comment:
I don't see that anything is asserted here after clicking the prerequisite. What is being tested?